application/app/Http/Controllers/Admin/AdminUsersController.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Models\AdminUser;
+
+class AdminUsersController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $query = AdminUser::query();
+        $name = $request->get('name', null);
+        $email = $request->get('email', null);
+        $authority = $request->get('authority', 'all');
+        $sort = $request->get('sort', 10);
+        $pageUnit = $request->get('page_unit', 10);
+
+        switch ($sort){
```

Comment:
```suggestion
        switch ($sort) {
```
スペース開けてください。他の部分も確認してください。